samples/msal-react-samples/b2c-sample/src/pages/Protected.jsx

```diff
@@ -22,9 +22,10 @@ const ProtectedContent = () => {
         if (!atsResponse && account && inProgress === InteractionStatus.None) {
             const request = {
                 ...loginRequest,
-                account: account
+                loginHint: account.username
             };
-            instance.acquireTokenSilent(request).then((response) => {
+
+            instance.ssoSilent(request).then((response) => {
```

Comment:
I don't think we want to invoke ssoSilent this often. Have you tried invoking `acquireTokenSilent` with `forceRefresh` set to `true`?

Reply:
I use the button "Edit profile" that call the "_instance.loginPopup(b2cPolicies.authorities.editProfile)_" to change the city from **Bari** to **Torino** as image below (in the claims i pass the city information)
<img width="391" alt="B2c_EditProfile" src="https://user-images.githubusercontent.com/3338265/176247229-1669d14f-af1e-4eab-b5ae-c19c52209d0a.png">

When I show the information of token, the city claim isn't the last changed (as you said I use _forceRefresh: true_)

<img width="1259" alt="result_request" src="https://user-images.githubusercontent.com/3338265/176247287-e157fee3-3525-4734-aa10-bcb8f9704ca0.png">

Reply:
I used in editprofile component the method "loginRedirect" and all work fine. With loginPopUp ther'is a problem in refresh token